binn/importance_network.py

```diff
@@ -97,6 +100,9 @@ def plot_complete_sankey(
         node_cmap: str = "Reds",
         edge_cmap: Union[str, list] = "Reds",
         savename="sankey.png",
+        width:int=1900,
```

Comment:
Formatting is inconsistent for this, should run Black

Type hints are also incomplete for the function signature